frontend/src/components/LandingPage/Navbar/Navbar.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { 
+    Navbar,
+    NavbarBrand,
+    Nav,
+    NavItem,
+    NavLink
+ } from 'reactstrap';
+
+function NavBar()
+{
+    //Navbar for github link, user input, and home page
```

Comment:
What do you think about removing this comment? Im not sure if its too helpful.

Reply:
Yeah, it's just a habit to write down comments and I agree it doesn't seem pretty useful